AlgebraInLean/Basic.lean

```diff
@@ -1 +1,49 @@
-def hello := "world"
\ No newline at end of file
+import Mathlib.Tactic
+
+namespace Defs
+  class Magma (G : Type*) where
+    op : G → G → G
+
+  -- FIXME: is this actually helpful/necessary
+  attribute [always_inline] Magma.op
```

Comment:
I'm pretty sure this doesn't do anything when people use `μ` defined below. I think we should use a `notation` instead of a `def` there, or just be done with it and remove this attribute.